src/main/java/mate/academy/rickandmorty/mapper/character/CharacterMapperImpl.java

```diff
@@ -0,0 +1,43 @@
+package mate.academy.rickandmorty.mapper.character;
+
+import mate.academy.rickandmorty.dto.character.CharacterDto;
+import mate.academy.rickandmorty.dto.character.CharacterFromApiDto;
+import mate.academy.rickandmorty.model.Character;
+import org.springframework.stereotype.Component;
+
+@Component
+public class CharacterMapperImpl implements CharacterMapper {
+
+    @Override
+    public CharacterDto toDto(Character character) {
+        if (character == null) {
+            return null;
+        }
+
+        CharacterDto characterDto = new CharacterDto();
+
+        characterDto.setId(character.getId());
+        characterDto.setExternalId(character.getExternalId());
+        characterDto.setName(character.getName());
+        characterDto.setStatus(character.getStatus());
+        characterDto.setGender(character.getGender());
+
+        return characterDto;
+    }
+
+    @Override
+    public Character toModel(CharacterFromApiDto character) {
+        if (character == null) {
+            return null;
+        }
+
+        Character character1 = new Character();
```

Comment:
```suggestion
    public Character toModel(ApiCharacterDto characterDto) {
        if (character == null) {
            return null;
        }

        Character character = new Character();
```
naming with numbers is  usually a bad practice